index.js

```diff
@@ -37,6 +36,7 @@ function Ciper(options) {
   //
   this.admins = options.admins || [];
   this.nodeType = options.nodeType || '';
+  this.isMaster = options.isMaster || false;
```

Comment:
can we make this a generic `type` property? we should have an array of valid `types` that the `options.type` would be checked against so that we arent limited to just master